### PR TITLE
Create PetOwner Model with doc comments and without tests

### PR DESCRIPTION
**Pull request by iBYarchitect**
## Summary
This PR is related to the issue #20. 
Sets the groundwork for displaying pet owner information in the app by creating a data model that can be easily used and manipulated in SwiftUI.

### Changes

- Add struct `PetOwner` with properties, such as `name`, `surname`, `id`, `avatar`, `phone`, `rating`, and `address`.

### CODE REVIEW

File: Woof/Woof/Model/PetOwner.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/**
+ A struct representing a pet owner.
+ This struct conforms to the Identifiable and Codable protocols, allowing it
+ to be easily used in SwiftUI and encoded/decoded from JSON.
+ */
+struct PetOwner: Identifiable, Codable {
```
Comment (NadzeyaShpakouskaya) [inline Woof/Woof/Model/PetOwner.swift:8]: nit: maybe `Owner` will be enough as a name for model?

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM, well done.

File: Woof/Woof/Model/PetOwner.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/**
+ A struct representing a pet owner.
+ This struct conforms to the Identifiable and Codable protocols, allowing it
+ to be easily used in SwiftUI and encoded/decoded from JSON.
+ */
```
Comment (dersim-davaod) [inline Woof/Woof/Model/PetOwner.swift:7]: Your documentation does not give any new information:
```suggestion
/**
A person who owns a pet (or several pets) and might be interested in finding a pet sitter for them.
 */
```

File: Woof/Woof/Model/PetOwner.swift [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    /// The first name of the pet owner.
+    var name: String
+    
+    /// The last name of the pet owner.
+    var surname: String
+    
+    /// The phone number of the pet owner.
+    var phone: String
+    
+    /// The optionl URL of the pet owner's avatar image.
```
Comment (dersim-davaod) [inline Woof/Woof/Model/PetOwner.swift:21]:
```suggestion
    /// The optional URL of the pet owner's avatar image.
```

Comment (dersim-davaod) [inline Woof/Woof/Model/PetOwner.swift:21]: nit: I see it's optional :) 

```suggestion
    /// URL that contains an avatar image of the pet owner.
```

File: Woof/Woof/Model/PetOwner.swift [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    var avatarUrl: URL?
+    
+    /// The address of the pet owner.
+    var address: String
+    
+    /// The rating of the pet owner from 0 to 5.
```
Comment (dersim-davaod) [inline Woof/Woof/Model/PetOwner.swift:27]: Am I allowed to put the number `-1` into the `rating` field? If so, how you can guarantee the documentation is correct?

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
Consider few minor improvements:

File: Woof/Woof/Model/PetOwner.swift (hunk 1 above)
Comment (iBYarchitect) [reply Woof/Woof/Model/PetOwner.swift:8]: Yes, sure. We have only one type of owner and no extra clarification is needed.

File: Woof/Woof/Model/PetOwner.swift (hunk 4 above)
Comment (iBYarchitect) [reply Woof/Woof/Model/PetOwner.swift:27]: Yes you can, I should add this validation check in the future in the view model layer. 
And i need to delete this part: 

> from 0 to 5

.

File: Woof/Woof/Model/PetOwner.swift (hunk 3 above)
Comment (iBYarchitect) [reply Woof/Woof/Model/PetOwner.swift:21]: Ok, i'll fix it

Comment (iBYarchitect) [reply Woof/Woof/Model/PetOwner.swift:21]: i'll fix it

File: Woof/Woof/Model/PetOwner.swift (hunk 2 above)
Comment (iBYarchitect) [reply Woof/Woof/Model/PetOwner.swift:7]: Ok, your suggestion looks clear. I'll change it.

**Review by julyakmita — APPROVED** [review]
Good job!
LGTM)

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**